src/bp-activity/classes/class-bp-activity-activity.php

```diff
@@ -1546,6 +1546,34 @@ public static function get_activity_comments( $activity_id, $left, $right, $spam
 			 */
 			if ( apply_filters( 'bp_use_legacy_activity_query', false, __METHOD__, $function_args ) ) {
 
+				$sql['select'] = "SELECT a.*, u.user_email, u.user_nicename, u.user_login, u.display_name{$fullname_select} ";
+				$sql['from']   = "FROM {$bp->activity->table_name} a, {$wpdb->users} u{$fullname_from} ";
+				$sql['where']  = "WHERE u.ID = a.user_id {$fullname_where} AND a.type = 'activity_comment' {$spam_sql} AND a.item_id = %d AND a.mptt_left > %d AND a.mptt_left < %d";
+				$sql['misc']   = "ORDER BY a.date_recorded ASC";
+
+				/**
+				 * Filters the MySQL From query for legacy activity comment.
+				 *
+				 * @since BuddyPress 1.5.0
```

Comment:
@dipskakadiya do the everywhere `BuddyPress 1.5.0` to `BuddyBoss 1.5.5`
```suggestion
				 * @since BuddyBoss 1.5.5
```